packages/ebayui-core/src/components/ebay-video/index.marko

```diff
@@ -39,14 +44,16 @@ $ const styleSize = {
         <subscribe to=window on-resize("handleResize")/>
     </if>
     <div class="video-player__container" no-update style=styleSize>
-        <video no-controls poster=thumbnail ...processHtmlAttributes(htmlInput)>
-            <for|src| of=sources>
-                <source ...src>
-            </for>
-            <for|track| of=tracks>
-                <track ...track>
-            </for>
-        </video>
+        <${nav ? "a" : null} ...processHtmlAttributes(nav || {})>
+            <video no-controls poster=thumbnail ...processHtmlAttributes(htmlInput)  key="video-element" class="video-player__video-element" muted=muted>
```

Comment:
`muted` should be passed in spread. If its not being passed, remove it from the `input` destructure. 

Also why is key being added and a new class which both seem unused.

Reply:
This is the code thats in main
https://github.com/eBay/evo-web/blob/main/packages/ebayui-core/src/components/ebay-video/index.marko#L42
Your new code is: 
```
<video no-controls poster=thumbnail ...processHtmlAttributes(htmlInput)  key="video-element" class="video-player__video-element" muted=muted>
```
So its not the same as in main, it needs to be reverted.